ukrainian/make-a-hole/go-to-sleep/go-to-sleep.md

```diff
@@ -0,0 +1,2 @@
+Прокинувшись від сну, головий герой спостерігає ту саму картину, розпачу не має меж.
+[спробую розважити себе переглядом кумендого серіалу](../tv-show/tv-show.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.